### PR TITLE
NO-JIRA: Adding some additional doco to the remote module

### PR DESCRIPTION
**Pull request by carnellj-genesys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +13,26 @@ module "AwsEventBridgeIntegration" {
 }
 ```
 
+The code above will create an AWS EventBridge integration. Genesys Cloud will create a partner event source in your AWS account. However, you will still need to create an event bridge for that partner event source. This can be done by creating by using the AWS terraform provider and then using the Terraform code below.
```
Comment (ronanwatkins) [inline README.md]: by creating by using -> by using

File: README.md [hunk 2]
```diff
@@ -13,10 +13,26 @@ module "AwsEventBridgeIntegration" {
...
+  name              = data.aws_cloudwatch_event_source.genesys_event_bridge.name
+  event_source_name = data.aws_cloudwatch_event_source.genesys_event_bridge.name
+}
+```
+
 ## Requirements
 
 | Name | Version |
-|------|---------|
+|------|---------|`
```
Comment (ronanwatkins) [inline README.md]: Is this backtick needed?

File: README.md (hunk 1 above)
Comment (carnellj-genesys) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (carnellj-genesys) [reply README.md]: Fixed.  Must of been in a wrong place when I did my typo.

**Pull request merged**